src/sentry/models/integrations/integration.py

```diff
@@ -67,9 +67,7 @@ def get_provider(self) -> IntegrationProvider:
         return integrations.get(self.provider)
 
     def delete(self, *args, **kwds):
-        with outbox_context(
-            transaction.atomic(router.db_for_write(OrganizationIntegration)), flush=False
-        ):
+        with outbox_context(transaction.atomic(using=None), flush=False):
```

Comment:
This should probably keep what it had before but add `using` as below:

`using=router.db_for_write(OrganizationIntegration)`